Microservices/Posts/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Stage 1: Build
+FROM maven:3.9.9-eclipse-temurin-21 AS builder
+WORKDIR /app
+COPY pom.xml .
+COPY src ./src
+RUN mvn clean package -DskipTests
+
+# Stage 2: Run
+FROM openjdk:21-jdk-slim
```

Comment:
we are using temurin distributor elsewhere in the project, will this cause any problems?

Reply:
Only profile has the different one. All the other ones use the openjdk